sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -569,7 +570,11 @@ private[hive] class HiveClientImpl(
 
   override def createTable(table: CatalogTable, ignoreIfExists: Boolean): Unit = withHiveState {
     verifyColumnDataType(table.dataSchema)
-    client.createTable(toHiveTable(table, Some(userName)), ignoreIfExists)
+    val ownerType =
+      table.properties.getOrElse(TableCatalog.PROP_OWNER_TYPE, PrincipalType.USER.name())
```

Comment:
when there is no owner type in table properties?

Reply:
the hive catalog's ownership is decided by this class as Database does,  it's simpler. For v2 commands, we have to do injection in our plans because we are not the catalog developer. As if we are, we also only need to inject ownership in that catalog's `createTable `